stream/stream.py

```diff
@@ -97,7 +97,11 @@ def __init__(self, fpath, mode='rb', **kwargs):
                 True by yielding a delimiter after reading each group.
             delimiter_cls (class): delimiter class.
         """
-        self._fd = gzip.open(fpath, mode)
+        if kwargs.get('gzip', True):
```

Comment:
It would be great if you could explain briefly about `gzip` argument in the `Keyword args` section of the `__init__`'s docstring.